packages/big-design/src/components/TableNext/Actions/styled.tsx

```diff
@@ -0,0 +1,25 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { FlexProps } from '../../Flex';
+import { withFlexedContainer } from '../../Flex/withFlex';
+
+export const StyledFlex = styled.div<FlexProps & { stickyHeader?: boolean }>`
+  ${withFlexedContainer()}
```

Comment:
I guess it really depends on the situation. If we extend via `styled(Flex)` we introduce a lot more props than just the `FlexProps`; this would introduce `BoxProps`, `PaddingProps`, `MarginProps`, & `DisplayProps`. If we do it the `styled.div<FlexProps>` way we can narrow the props to what we want without having to filter out the unwanted props.